include/zephyr/devicetree.h

```diff
@@ -17,9 +17,12 @@
 #define DEVICETREE_H
 
 #include <devicetree_generated.h>
-
 #include <zephyr/sys/util.h>
 
+#ifndef _ASMLANGUAGE
+#include <stdint.h>
+#endif
+
```

Comment:
```suggestion
#ifndef _LINKER
#include <stdint.h>
#endif

```

also, move after devicetree_generated.h

Reply:
Ok, will do, thanks!